egs2/qasr_tts/adapt.sh

```diff
@@ -0,0 +1,31 @@
+. local/pretrained.sh

```

Comment:
Cool procedure, can we have a better documentation on this script (e.g., maybe in RESULTS.md or README.md of this recipe)

Reply:
Okay I will add a README